wurst/api/cli.py

```diff
@@ -0,0 +1,66 @@
+# encoding: utf8
+
+from __future__ import unicode_literals
+
+import sys
+from collections import namedtuple
+
+from wurst.core.models.issues import IssueType, Priority, Status
+
+if sys.version_info[0] < 3:
+    import ushlex as shlex
+else:
+    import shlex
+
+
+NOUN_CLASSES = [IssueType, Status, Priority]
+
+# Stub, replace with a real Command class
+Command = namedtuple("Command", "slug verbs")
+
+wurst = Command(slug="wurst", verbs=["wurst"])  # mostly ignored
+create = Command(slug="create", verbs=["create", "new", "c"])
+
+COMMANDS = [wurst, create]
+
+
+class Context(object):
+    """
+    A Context contains the hot words for the current environment (project etc.) mapped to their
+    model counterparts.
+    """
+
+    def __init__(self):
+        self.terms = self._get_terms()
+
+    def _get_terms(self):
```

Comment:
This could be a cached property called `terms`. (I think Django might supply a `@cached_property` decorator?)